terraform/environments/fetureflag/main.tf

```diff
@@ -0,0 +1,135 @@
+# ----------------------------------------------------------------------------------------------------------------------
```

Comment:
We don't need this in the commit....lets keep it local

Reply:
It will create problems if CL people creates multiple envs

Reply:
you can merge now